app/src/main/java/org/p2p/wallet/home/onofframp/OnOffRampNavigator.kt

```diff
@@ -0,0 +1,68 @@
+package org.p2p.wallet.home.onofframp
+
+import androidx.fragment.app.Fragment
+import org.p2p.wallet.common.NavigationDestination
+import org.p2p.wallet.common.NavigationStrategy
+import org.p2p.wallet.home.addmoney.ui.AddMoneyDialog
+import org.p2p.wallet.home.onofframp.ui.OnOffRampCountrySelectionFragment
+import org.p2p.wallet.settings.interactor.SettingsInteractor
+import org.p2p.wallet.striga.StrigaFragmentFactory
+import org.p2p.wallet.striga.offramp.ui.StrigaOffRampFragment
+import org.p2p.wallet.utils.replaceFragment
+import org.p2p.wallet.utils.replaceFragmentForResult
+
+class OnOffRampNavigator(
```

Comment:
Just helps to navigate, because now on/off ramp screens should be opened only if user has chosen country